web/src/composebox_typeahead.js

```diff
@@ -411,7 +411,7 @@ export function broadcast_mentions() {
     if (compose_state.get_message_type() === "private") {
         wildcard_mention_array = ["all", "everyone"];
     } else if (compose_validate.stream_wildcard_mention_allowed()) {
-        wildcard_mention_array = ["all", "everyone", "stream", "channel", "topic"];
+        wildcard_mention_array = ["all", "everyone", "channel", "topic"];
```

Comment:
We probably don't want to cut `stream` from the typeahead just yet -- we should defer that until we're ready to swap the strings.

Reply:
Okay. I've added a final commit to this series that adds some helper functions to `util.ts` for checking/converting "channel" to "stream" to be used for implementing these (and other) updates related to the rename prior to the main user-facing strings change.

The plan would be to then switch these to do the reverse when we're at the final stages of the rename process: check/convert "stream" to "channel".

And in that commit, we use those to add the "stream" wildcard mention syntax to the message content if the "channel" option is selected from the composebox typeahead. If a user types out the correct mention syntax with "channel", then that is used in the message content and functions as a stream wildcard mention is expected to when the message is sent.